highlight_view.py

```diff
@@ -103,17 +104,37 @@ def on_lint_result(buffer_id, linter_name, **kwargs):
         view, linter_name, errors_for_the_gutter)
 
     for view in views:
+        vid = view.id()
+
+        if (
+            persist.settings.get('highlights.start_hidden') and
+            vid not in State['quiet_views'] and
+            vid not in State['views']
+        ):
+            State['quiet_views'].add(vid)
+
+        if vid not in State['views']:
+            State['views'].add(vid)
+
         draw(
             view,
             linter_name,
             highlight_regions,
             gutter_regions,
             protected_regions,
-            idle=(view.id() in State['idle_views']),
-            quiet=(view.id() in State['quiet_views'])
+            idle=(vid in State['idle_views']),
+            quiet=(vid in State['quiet_views'])
         )
 
 
+class GarbageController(sublime_plugin.EventListener):
```

Comment:
I'm not sure I'd call it GarbageController, garbage is a loaded term.. To really complain about it I should come up with a different term. `ViewListCleanupController`?

Reply:
When @braver complains about such names it's usually a joke. 🤔